### ISSUE TITLE
Eviction in version "v1" cannot be handled as a Eviction

### ISSUE THREAD

**Issue by dry4ng** [kubernetes-sigs/descheduler#977]
**What version of descheduler are you using?**

descheduler version: 1.25

**Does this issue reproduce with the latest release?**

yes

**Which descheduler CLI options are you using?**

none

**Please provide a copy of your descheduler policy config file**

```deschedulerPolicy:
  strategies:
    RemoveDuplicates:
      enabled: false
    RemovePodsViolatingNodeTaints:
      enabled: false
    RemovePodsViolatingNodeAffinity:
      enabled: false      
    RemovePodsViolatingInterPodAntiAffinity:
      enabled: false
    LowNodeUtilization:
      enabled: false
    RemovePodsViolatingTopologySpreadConstraint:
      enabled: true
      params:
        includeSoftConstraints: true
```

**What k8s version are you using (`kubectl version`)?**

1.21 (GKE)

**What did you do?**

I have upgraded to v1.25 and started seeing these errors with every descheduler run:

```
E1010 18:10:02.963839       1 evictions.go:141] "Error evicting pod" err="Eviction in version \"v1\" cannot be handled as a Eviction: no kind \"Eviction\" is registered for version \"policy/v1\" in scheme \"k8s.io/kubernetes/pkg/api/legacyscheme/scheme.go:30\"" pod="prod/my-pod-name" reason=""
```

**Comment by damemi** [comment]
Hi @dry4ng, this is the same error as in https://github.com/kubernetes-sigs/descheduler/issues/972#issuecomment-1270284469, please see that comment.

tl;dr is that descheduler v0.25 is not supported on k8s 1.21, the latest descheduler version you can use is v0.23
/close
(closing as duplicate)
